core/server/config/overrides.json

```diff
@@ -6,6 +6,7 @@
         "helperTemplates": "core/server/helpers/tpl/",
         "adminViews": "core/server/web/admin/views/",
         "defaultViews": "core/server/views/",
+        "defaultSettings": "core/server/settings/",
```

Comment:
There is no folder `core/server/settings` 🤔 There is only a file in `core/settings/..`.

Reply:
Changed to `core/server/settings` see [here](https://github.com/TryGhost/Ghost/pull/9534#discussion_r179421528)